test/TestUtil.js

```diff
@@ -656,4 +656,11 @@ exports.waitForConnectionCount = async (client, connectionCount) => {
         expect(getConnectionsFn().length).to.be.equal(connectionCount);
     });
 };
+/**
+ * This function converts hours to seconds
+ */
+exports.GetTimeOffsetByHour = (hour) => {
```

Comment:
```suggestion
exports.getTimeOffsetByHour = (hour) => {
```

not sure this method is necessary but, no harm in having more methods. they increase readability.

Reply:
When I compare with java side, I decide to create a method for more readability.

```
OffsetDateTime.of(LocalDate.of(1938, 11, 10), LocalTime.of(9, 5, 10, 123456789),
                        ZoneOffset.ofHours(18)),
```